src/main/java/frc/robot/vision/multivisionsources/MultiVisionSources.java

```diff
@@ -0,0 +1,64 @@
+package frc.robot.vision.multivisionsources;
```

Comment:
there isn't any point for having private field or function in that class. Everything should be accessible through `MultiAprilTagVisionSources` (and potentially `MultiObjectDetectionVisionSources` or any multivisionsources that we would want to add in the future), even if we don't use it.